Common/node/lcrypto.js

```
@@ -106,3 +111,24 @@ exports.decrypt = function(data) {
     ret += cipher.final("utf8");
     return ret;
 }
+
+exports.sign = function(data) {
+    if (!data) {
+        logger.error("Error signing " + data);
```

this should really throw an Error